llvm/include/llvm/CodeGen/LiveIntervals.h

```diff
@@ -50,445 +51,492 @@ class raw_ostream;
 class TargetInstrInfo;
 class VirtRegMap;
 
-  class LiveIntervals : public MachineFunctionPass {
```

Comment:
`clang-format` formats almost entire class after modifying this line.

Reply:
can you split out the formatting change and submit that one first?